### PR TITLE
media-libs/kcolorpicker-0.2.0 and media-libs/kimageannotator-0.6.0 bumps

### PR DESCRIPTION
**Pull request by dflogeras**

### CODE REVIEW

**Comment by dflogeras** [comment]
Upstream changed the tests/ to require running from within that directory.  Please review my src_test()

I also tried (under Sam's direction):
    local myctestargs=(
       --test-dir tests
    )
    virtx cmake_src_test

To no avail.

File: media-libs/kcolorpicker/kcolorpicker-0.2.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# Copyright 2020-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```
Comment (thesamesam) [inline media-libs/kcolorpicker/kcolorpicker-0.2.0.ebuild:4]: 8?

File: media-libs/kimageannotator/kimageannotator-0.6.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+src_configure() {
+	local mycmakeargs=(
+		-DBUILD_TESTS=$(usex test)
+	)
+	cmake_src_configure
+}
+
+src_test() {
+	BUILD_DIR="${BUILD_DIR}/tests" virtx cmake_src_test
```
Comment (thesamesam) [inline media-libs/kimageannotator/kimageannotator-0.6.0.ebuild:48]: This seems okay, although I wonder if you can do a `cd` instead.

Comment (dflogeras) [reply media-libs/kimageannotator/kimageannotator-0.6.0.ebuild:48]: Unless I misunderstand, I just tried this and it still did not run tests:

```
src_test() {
        cd ${BUILD_DIR}/tests
        virtx cmake_src_test
}
```

Comment (juippis) [reply media-libs/kimageannotator/kimageannotator-0.6.0.ebuild:48]: Something happens here with your current implementation,
```
100% tests passed, 0 tests failed out of 49
```

**Review by juippis — APPROVED** [review]

**Comment by dflogeras** [comment]
Oh, I knew it got the desired result, just wondered if anyone had a more elegant/gentoo way of doing it.

All good, thanks!

**Pull request closed without merging**